### PR TITLE
Configuration mas faa

### PR DESCRIPTION
**Pull request by jakob123100**
Code cleanup and preparation for full system test.

### CODE REVIEW

File: src/final_score_calculator/calculator.py
Comment (grufe) [inline src/final_score_calculator/calculator.py:1]: Check pylint

final_score_calculator\calculator.py:15:39: C0303: Trailing whitespace (trailing-whitespace)
final_score_calculator\calculator.py:45:0: C0303: Trailing whitespace (trailing-whitespace)
final_score_calculator\calculator.py:53:0: C0303: Trailing whitespace (trailing-whitespace)
final_score_calculator\calculator.py:15:8: W0621: Redefining name 'dependencies' from outer scope (line 66) (redefined-outer-name)
final_score_calculator\calculator.py:46:8: W0621: Redefining name 'dependency' from outer scope (line 62) (redefined-outer-name)

File: src/frontend_api.py
Comment (grufe) [inline src/frontend_api.py:1]: - Rewrite docstrings to include arguments and return description
- Note for future development: Many input arguments in functions. Should requirements be its own data structure?

Check pylint

frontend_api.py:74:60: C0303: Trailing whitespace (trailing-whitespace)
frontend_api.py:80:70: C0303: Trailing whitespace (trailing-whitespace)
frontend_api.py:63:0: R0913: Too many arguments (6/5) (too-many-arguments)
frontend_api.py:77:16: R1732: Consider using 'with' for resource-allocating operations (consider-using-with)

File: src/tests/test_frontend_api.py
Comment (grufe) [inline src/tests/test_frontend_api.py:1]: In *test_integer_value()* and *test_decimal_value()* the function *check_input_arguments()* is tested with the same values in one case. One of these checks could be removed.

Note for future: Consider boundary value testing?

File: src/tests/test_util.py
Comment (grufe) [inline src/tests/test_util.py:1]: - No assert in *test_check_token_usage()* (line: 98)

Check pylint

File: src/util.py
Comment (grufe) [inline src/util.py:1]: Check function return declarations
Check pylint

util.py:90:0: C0303: Trailing whitespace (trailing-whitespace)
util.py:104:0: C0303: Trailing whitespace (trailing-whitespace)
util.py:105:0: C0301: Line too long (144/100) (line-too-long)
util.py:1:0: C0114: Missing module docstring (missing-module-docstring)
util.py:102:4: R1705: Unnecessary "else" after "return", remove the "else" and de-indent the code inside it (no-else-return)

**Review by grufe — APPROVED** [review]
Approved for prototype version with some comments.

Check comments for each individual file for more detailed info

**Review by Bacilika — APPROVED** [review]
looks good

File: src/MAS.py
[the remark this answers is not in the document]
Comment (jakob123100) [reply src/MAS.py:1]: Fixed

File: src/calculate_dependencies.py
[the remark this answers is not in the document]
Comment (jakob123100) [reply src/calculate_dependencies.py:1]: Fixed all exept todo issues

File: src/frontend_api.py
Comment (jakob123100) [reply src/frontend_api.py:1]: Fixed

File: src/tests/test_frontend_api.py
Comment (jakob123100) [reply src/tests/test_frontend_api.py:1]: Fixed

File: src/tests/test_util.py
Comment (jakob123100) [reply src/tests/test_util.py:1]: fixed

File: src/util.py
Comment (jakob123100) [reply src/util.py:1]: fixed

File: src/cmd_ui.py
[the remark this answers is not in the document]
Comment (jakob123100) [reply src/cmd_ui.py:1]: Fixed ?

**Pull request merged**